java10/scripts.js

```diff
@@ -0,0 +1,145 @@
+var newGameBtn = document.getElementById('js-newGameButton'),
+    newGameElem = document.getElementById('js-newGameElement'),
+    pickElem = document.getElementById('js-playerPickElement'),
+    resultsElem = document.getElementById('js-resultsTableElement');
+
```

Comment:
nie deklaruj zmiennych po przecinku - to powszechnie uważane jest za anti-pattern.